.github/actions/verify-submodule-pull-request/action.yml

```diff
@@ -15,11 +15,9 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Setup and Print PR Title
+    - name: Setup Scripts
       shell: pwsh
       run: |
-        Write-Output "Pull Request Title: ${{ inputs.title }}"
```

Comment:
I actually had a problem with this (or something similar) way back, when the PR title contained text that was formatted as `code`: https://github.com/Lombiq/GitHub-Actions/pull/43

Reply:
So you and Dávid did some script injection? 
![image](https://user-images.githubusercontent.com/1976647/209479193-a20a76ac-f222-4cc8-8df8-a4ce57487482.png)